### PR TITLE
[RET-1463] Update import package for new et-data-model version (0.0.10)

### PR DESCRIPTION
**Pull request by jsinghm13**
https://tools.hmcts.net/jira/browse/RET-1463

Update import package for new et-data-model version (0.0.10)

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/et/syaapi/service/CaseService.java [hunk 1]
```diff
@@ -78,7 +78,7 @@ public CaseDetails createCase(String authorization, String caseType, String even
         var ccdCase = ccdApiClient.startForCaseworker(
             authorization,
             s2sToken,
-            userDetails.getId(),
+            "123456",
```
Comment (vinothon84) [inline src/main/java/uk/gov/hmcts/reform/et/syaapi/service/CaseService.java:81]: Can you change this to userDetails.getId() ?

File: src/main/java/uk/gov/hmcts/reform/et/syaapi/service/CaseService.java [hunk 2]
```diff
@@ -92,7 +92,7 @@ public CaseDetails createCase(String authorization, String caseType, String even
         return ccdApiClient.submitForCaseworker(
             authorization,
             s2sToken,
-            userDetails.getId(),
+            "123456",
```
Comment (vinothon84) [inline src/main/java/uk/gov/hmcts/reform/et/syaapi/service/CaseService.java:95]: Can you change this to userDetails.getId() ?

File: src/main/java/uk/gov/hmcts/reform/et/syaapi/service/CaseService.java (hunk 1 above)
Comment (jsinghm13) [reply src/main/java/uk/gov/hmcts/reform/et/syaapi/service/CaseService.java:81]: Fixed

File: src/main/java/uk/gov/hmcts/reform/et/syaapi/service/CaseService.java (hunk 2 above)
Comment (jsinghm13) [reply src/main/java/uk/gov/hmcts/reform/et/syaapi/service/CaseService.java:95]: Fixed

**Review by vinothon84 — APPROVED** [review]

**Pull request closed without merging**